src/main.rs

```diff
@@ -174,11 +176,44 @@ fn get_config_value(name: &str) -> Result<String, String> {
     }
 }
 
+// sysctl: a YAML-formatted string of sysctl options eg "{ 'kernel.max_pid': 1337 }"
+fn create_sysctl<T: Write>(sysctl: String, f: &mut T) -> Result<usize, String> {
+    let deserialized_map: BTreeMap<String, String> =
+        serde_yaml::from_str(&sysctl).map_err(|e| e.to_string())?;
+
+    let mut bytes_written = 0;
+    for (key, value) in deserialized_map {
+        bytes_written += f.write(&format!("{}={}\n", key, value).as_bytes())
+            .map_err(|e| e.to_string())?;
+    }
+    Ok(bytes_written)
+}
+
 fn config_changed() -> Result<(), String> {
     check_for_upgrade()?;
+    check_for_sysctl()?;
```

Comment:
I'm not sure if I have to add this to the initial deployment before volume start.  I'll check